build_for_ultimaker.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+#
+# SPDX-License-Identifier: AGPL-3.0+
+#
+# Copyright (C) 2019 Ultimaker B.V.
+#
+# SPDX-License-Identifier: LGPL-3.0+
+
+set -eu
+
+ARCH="armhf"
+
+SRC_DIR="$(pwd)"
+RELEASE_VERSION="${RELEASE_VERSION:-999.999.999}"
+DOCKER_WORK_DIR="/build"
+BUILD_DIR_TEMPLATE="_build_${ARCH}"
+BUILD_DIR="${BUILD_DIR:-${SRC_DIR}/${BUILD_DIR_TEMPLATE}}"
+
+run_env_check="yes"
+run_linters="yes"
+run_tests="yes"
+
+# Make sure to pass an empty argument to make_docker, else any arguments passed to build_for_ultimaker is passed to make_docker instead!
+. ./make_docker.sh ""
+
+env_check()
+{
+    run_in_docker "./docker_env/buildenv_check.sh"
+}
+
+run_build()
+{
+    run_in_docker "./build.sh" "${@}"
+}
+
+deliver_pkg()
+{
+    run_in_docker chown -R "$(id -u):$(id -g)" "${DOCKER_WORK_DIR}"
+
+    cp "${BUILD_DIR}/"*".deb" "./"
+}
+
+run_tests()
+{
+    echo "Testing!"
+    # These tests should never fail! See .gitlab-ci.yml
+    ./run_complexity_analysis.sh || echo "Code Complexity Analysis Failed!"
+    ./run_dead_code_analysis.sh || echo "Dead Code Analysis Failed!"
+    ./run_style_analysis.sh || echo "Style Analysis Failed!"
+    ./run_mypy.sh || echo "MYPY Analysis Failed!"
+    ./run_check_material_profiles.sh || echo "Material Profile Check Failed!"
+    ./run_check_material_profiles_new_with_lxml.sh || echo "Material Profile Check with lxml Failed!"
```

Comment:
You'll need only one of these, preferably the lxml version if you have lxml on your build environment.